src/components/MoneyRequestHeader.tsx

```diff
@@ -131,7 +117,7 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
                     threeDotsAnchorPosition={styles.threeDotsPopoverOffsetNoCloseButton(windowWidth)}
                     report={{
                         ...report,
-                        ownerAccountID: lodashGet(parentReport, 'ownerAccountID', null),
+                        ownerAccountID: parentReport?.ownerAccountID,
```

Comment:
Let's default to 0?

Reply:
@MonilBhavsar I took a look at the code that will use this prop and it's safe to pass 0, but IMO it would be safer to pass undefined in case the current logic gets changed. What do you think?

Reply:
Yes, NAB